doc/Interns/zhaonancy/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 5, 2020

```

Comment:
Sorry, please correct to add the time too!

Reply:
Made the correction!